migrations/20190930-extend_objects_to_mediumtext-builds.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable new-cap */
+
+'use strict';
+
+const prefix = process.env.DATASTORE_SEQUELIZE_PREFIX || '';
+const table = `${prefix}builds`;
+
+module.exports = {
+    up: async (queryInterface, Sequelize) => {
+        await queryInterface.sequelize.transaction(async (transaction) => {
+            await queryInterface.changeColumn(table, 'meta', Sequelize.TEXT('medium'),
+                { transaction }
+            );
+        });
+
+        await queryInterface.sequelize.transaction(async (transaction) => {
+            await queryInterface.changeColumn(table, 'parameters', Sequelize.TEXT('medium'),
```

Comment:
it looks like we are not using this column at all. Better to remove this in a future PR. I will drop a separate issue for this.

Reply:
https://github.com/screwdriver-cd/screwdriver/issues/1774